docs/components/concepts/job-workers.md

```diff
@@ -48,15 +46,15 @@ To find a job to work on, the worker must poll again for available jobs. This le
 
 This is expensive in terms of resource usage, because both the worker and the server are performing a lot of unproductive work. Zeebe supports **long polling** for available jobs to better utilize resources.
 
-With **long polling**, a request will be kept open while no jobs are available. The request is completed when at least one job becomes available.
+With **long polling**, a request is kept open while no jobs are available. The request is completed when at least one job becomes available.
 
-**Long Polling** is set during [job activation with the parameter `request-timeout`](../../apis-tools/grpc.md#activatejobs-rpc).
+**Long polling** is set during [job activation with the parameter `request-timeout`](../../apis-tools/grpc.md#activatejobs-rpc).
 
-### Job queueing
+### Job queuing
 
 Zeebe decouples creation of jobs from performing the work on them. It is always possible to create jobs at the highest possible rate, regardless if there is a job worker available to work on them. This is possible because Zeebe queues jobs until workers request them.
 
-This increases the resilience of the overall system. Camunda 8 is highly available so job workers don't have to be highly available. Zeebe queues all jobs during any job worker outages, and progress will resume as soon as workers come back online.
+This increases the resilience of the overall system. Camunda 8 is highly available so job workers don't have to be highly available. Zeebe queues all jobs during any job worker outages, and progress resumes as soon as workers come back online.
```

Comment:
"as soon as workers come back online" -- what does this mean? Can we word this differently?

Reply:
You could say "as soon as workers are available," as this might be clearer in context.